x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/output_form_remote_es.tsx

```diff
@@ -332,7 +334,7 @@ export const OutputFormRemoteEsSection: React.FunctionComponent<Props> = (props)
                             followerIndex: (
                               <EuiCode>
                                 fleet-synced-integrations-ccr-
-                                {inputs.nameInput.props.value || '<output name>'}
+                                {snakeCase(inputs.nameInput.props.value) || '<output name>'}
```

Comment:
small fix to use a snake case name in ccr index e.g. `Remote ES demo -> remote_es_demo` 
<img width="791" alt="image" src="https://github.com/user-attachments/assets/3ec35d5e-06c5-4957-ad10-f76feaf1e794" />